packages/airnode-utilities/test/e2e/gas-oracle.feature.ts

```diff
@@ -103,6 +104,11 @@ const processBlockData = async (
   }
 };
 
+const resetAnvil = async () => {
+  spawn("bash", ["-c", 'killall anvil;']);
+  await new Promise(f => setTimeout(f, 1000));
```

Comment:
Ideally starting and stopping Anvil could be managed in code or handled more intelligently.